### ISSUE TITLE
Create a logo with text based on the existing logo

### ISSUE THREAD

**Issue by joshka** [ratatui-org/ratatui#656]
The logo on the website and various places currently just uses the default heading font. It would be nice if we could find something that looks a bit more modern to use when writing the word Ratatui and incorporate that into the logo design.

<img width="207" alt="image" src="https://github.com/ratatui-org/website/assets/381361/ada1a71f-5179-47e1-b821-6c63dc9e2b22">

There is a big transparent logo available as a starting point.

![ratatui-transparent](https://github.com/ratatui-org/ratatui/assets/381361/c433e835-20e9-4c86-adf7-61a419833f16)

Pinging @nawok to see if they want to jump in on this, but otherwise it's open to anyone that has some good taste and skills when it comes to font-picking.

**Comment by nawok** [comment]
Hi @joshka,

In the [initial logo discussion](https://github.com/ratatui-org/ratatui/discussions/86#discussioncomment-5286367) I also provided the wordmark in dot matrix printer style.

Do you suggest to try out a different design direction?

**Comment by joshka** [comment]
This looks ok...

![image](https://github.com/ratatui-org/ratatui/assets/381361/8cd993eb-e14b-4200-9784-2dd63219e43e)

![image](https://github.com/ratatui-org/ratatui/assets/381361/d993fdea-d4de-45a9-924e-d646462aca1a)

**Comment by nawok** [comment]
Hm… The contrast looks really low. I understand the goal better now and I will play around with other options.

Best regards,
Pavel

**Comment by joshka** [comment]
Yep. I think perhaps the letter pixels need to be a little larger - I didn't spend too much time on this - just resize without interpolation.

Also it's resized to 40 pixels high. When I resized the logo down from the 1000px version in gimp it came out pretty janky, so I went to 50 instead and let the browser scale down from that. The problem though is that this makes the pixels fuzzy from something that looked sharp. It would be nice to make this actual size right, but I think this might take re-positioning all the pixels manually.

**Comment by orhun** [comment]
Come to think of it, do we give credit to @nawok about the logo in the docs/repo anywhere? 🐻

**Comment by joshka** [comment]
> Come to think of it, do we give credit to @nawok about the logo in the docs/repo anywhere? 🐻

Yep - at the bottom of the github README
